src/main/java/com/google/api/codegen/csharp/CSharpDiscoveryContext.java

```diff
@@ -188,6 +303,8 @@ public SampleInfo getSampleInfo(Method method) {
     String serviceTypeName = packageName + "Service";
     String serviceVarName = CSharpContextCommon.s_underscoresToCamelCase(packageName) + "Service";
     String methodName = CSharpContextCommon.s_underscoresToPascalCase(getSimpleName(rawMethodName));
+    boolean hasRequestField = hasRequestField(method);
+    String requestFieldName = "content";
```

Comment:
Is the aim to vary this somewhere based on the discovery doc? At the moment it might as well be a constant, unless I've missed something.